### DISCUSSION TITLE
How to run netbeansRCP app in production without nbm:run-platform

### DISCUSSION THREAD

**Discussion by guilherme-oliveria** [apache/netbeans#4571] [General]
I migrated my application that was previously built by apache ANT on platformRCP to maven following these steps [tutorial](https://netbeans.apache.org/tutorials/nbm-maven-quickstart_en_BR.html), now I need to make the application available in production, I have a . exe that downloads updates using getdown [getdown](https://github.com/rockfireredmoon/getdown-maven-plugin), it maps clashPath and before it declared main as org.netbeans.modules.apisupport.jnlplauncher.Main, but with maven build it doesn't works, which mainClass should I use ? or does it only work the .exe that the nbm plugin generates in the target folder?

**Comment by guilherme-oliveria** [comment]
is org.netbeans.Main of org-netbeans-boostrap

**Comment by oyarzun** [comment]
@guilherme-oliveria after building the parent project, it should build a zip of the RCP application which would be located in the application project target dir. E.g. based on the tutorial. `/path/to/MavenPlatformWordApp/application/target`
